Upstream: no, Exherbo-specific
Reason: attempts to load kakrc from /usr/${CHOST}/bin/../share/kakrc, which makes no sense

From f20ba7867eb2128f528c99a9f98fc3468ed7eef4 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Sun, 26 Jul 2015 04:02:32 -0400
Subject: [PATCH] Fixes the location kakoune tries to load the system kakrc
 from

---
 src/main.cc | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/main.cc b/src/main.cc
index 6a92f9f..0fd789b 100644
--- a/src/main.cc
+++ b/src/main.cc
@@ -34,11 +34,7 @@ using namespace Kakoune;
 
 String runtime_directory()
 {
-    String bin_path = get_kak_binary_path();
-    auto it = find(reversed(bin_path), '/');
-    if (it == bin_path.rend())
-        throw runtime_error("unable to determine runtime directory");
-    return StringView{bin_path.begin(), it.base()-1} + "/../share/kak";
+    return "/usr/share/kak";
 }
 
 static void write(int fd, StringView str)
-- 
2.4.6

